### ISSUE TITLE
Shortcuts change on keyboard layout change

### ISSUE THREAD

**Issue by max-sixty** [ianyh/Amethyst#821]
Thank you so much for such an awesome tool. It's made measurably more productive than using Mirror, and cut the amount I need to use the mouse by ~1/3. I have one issue which makes switching layouts difficult:

## System

 - **Amethyst version**:
 - **macOS version**:

## What's the problem?

When I move from US to Dvorak, the shortcuts don't move - they're stuck on the 'physical' keys 

## How can it be reproduced?

- Set keyboard to Dvorak
- Set a shortcut (using the application, not dotfile)
- Change back to US
- Observe the shortcut changes

## Screenshots

On dvorak: 

![image](https://user-images.githubusercontent.com/5635139/55754539-e0e0f700-5a1a-11e9-9fb5-a09b11099766.png)

When switching to US:
![image](https://user-images.githubusercontent.com/5635139/55754591-079f2d80-5a1b-11e9-87d8-57e261b598e8.png)

## Similar issues

I saw a couple of somewhat-related issues, although  closed and not exactly the same. Referenced here:
https://github.com/ianyh/Amethyst/issues/352
https://github.com/ianyh/Amethyst/issues/12

**Comment by joefromct** [comment]
I have this problem too on my macbook pro.... and for that reason can only use amethyst when plugged into my usb hub with a [keyboard that has dvorak built-in](https://www.amazon.com/Mechanical-Keyboard-Keycaps-Switches-Mx-Blue/dp/B00OFM51L2).  I'll leave it on US keyboard layout and let the keyboard handle the dvorak. 

Not a fix, but maybe something to think about.

**Comment by max-sixty** [comment]
Yes, I do that when at my desk with Kinesis Advantage. But still an issue when on the mac keyboard

**Comment by ZioKli** [comment]
This is infuriating as a laptop user. I have an external keyboard which I use 80% or more of the time with workman baked in as the layout so I moved my hotkeys around to make sense for that. But that 20% of the time with the software workman layout has everything thrown around and makes it impossible to use effectively. Please just provide a toggle so that we can opt out of the remapping. I can see why it's useful when only looking at software layouts, but I have a feeling that the people inclined to use a tiling window manager are more likely than average to use external keyboards and non-qwerty layouts.

**Comment by masar3141** [comment]
any plan to fix this?

**Comment by ianyh** [comment]
I'm not sure I have time for it, but if someone wants to implement it I would accept it.
